### PR TITLE
Several fixes

### PR DESCRIPTION
**Pull request by fernandog**
@medariox @ratoaq2 pycharm that fixed this indentations byut travis is complaining...so who is right?

UPDATE: i had to manually fix it

### CODE REVIEW

File: medusa/providers/torrent/html/cpasbien.py [hunk 1]
```diff
@@ -72,8 +72,8 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
                 if mode != 'RSS':
                     logger.log('Search string: {search}'.format
                                (search=search_string), logger.DEBUG)
-                    search_url = self.url + '/recherche/' + search_string.replace('.', '-').replace(' ',
-                                                                                                    '-') + '.html,trie-seeds-d'
+                    search_url = self.url + '/recherche/' + \
+                        search_string.replace('.', '-').replace(' ', '-') + '.html,trie-seeds-d'
```
Comment (fernandog) [inline medusa/providers/torrent/html/cpasbien.py]: ok, this one was ugly

**Comment by fernandog** [comment]
@medariox thanks

**Comment by fernandog** [comment]
approve?

**Comment by medariox** [comment]
I'm not done yet.

**Pull request merged**